src/stories/components/wizard.stories.js

```diff
@@ -50,7 +50,7 @@ storiesOf('Components', module)
                                 return (
 
                                     <React.Fragment>
-                                        <Deprecated link='https://patternfly-react.surge.sh/patternfly-4/components/wizard/'/>
+                                        <Deprecated link='https://www.patternfly.org/v4/documentation/core/components/wizard'/>
```

Comment:
Let's update this one to use the react version instead of the html version: https://www.patternfly.org/v4/documentation/react/components/wizard/

Reply:
👍